x/tx/signing/context.go

```diff
@@ -301,6 +301,11 @@ func (c *Context) makeGetSignersFunc(descriptor protoreflect.MessageDescriptor)
 				}
 				return arr, nil
 			}
+		case protoreflect.BytesKind:
+			fieldGetters[i] = func(msg proto.Message, arr [][]byte) ([][]byte, error) {
+				addrBz := msg.ProtoReflect().Get(field).Bytes()
+				return append(arr, addrBz), nil
+			}
```

Comment:
the string case checks for `IsList`. given that `repeated bytes` is valid protobuf, why don't we do that here?

Reply:
We don't have 2D array bytes signers atm, but we can align the behavior.